bacon/controller/CoreLogic.swift

```diff
@@ -49,6 +50,25 @@ class CoreLogic: CoreLogicInterface {
             description=\(description) location=\(location as Optional).
             """)
         try transactionManager.saveTransaction(currentTransaction)
+        guard let prediction = prediction else {
+            // No prediction was used, hence no need to check whether prediction is accepted
+            return
+        }
+        if doAcceptPrediction(transaction: currentTransaction, prediction: prediction) {
+            do {
+                try predictionManager.savePrediction(prediction)
+            } catch {
+                // Failure in prediction should be resolved internally, as it is not known at all by the user
+                log.warning("CoreLogic failed saving prediction")
```

Comment:
Should we do anything other than just log?

Reply:
im not sure... my personal opinion is that
1) it should not be propagated up because user has completely no idea of this prediction thing going on
2) it should also not affect the normal functioning of the app, because it's a secondary feature
I don't know what else can be done